lib/verdict.rb

```diff
@@ -20,7 +20,11 @@ def repository
   end
 
   def discovery
-    Dir[File.join(Verdict.directory, '**', '*.rb')].each { |f| require f } if @directory
+    Dir[File.join(Verdict.directory, '**', '*.rb')].each { |f| load f } if @directory
```

Comment:
Is verdict rails specific? If so it'd be better to constantize strings or use ActiveSupport::Dependencies to load this file in some other mechanism so that it just does the right thing underneath.